WalletWasabi.Fluent/Views/MainView.axaml

```diff
@@ -56,6 +56,15 @@
               HorizontalAlignment="Stretch" VerticalAlignment="Stretch"
               HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch"
               MaxContentWidth="800" MaxContentHeight="700">
+      <c:Dialog.Styles>
+        <Style Selector="c|Dialog /template/ Panel#PART_DialogHost">
+          <Setter Property="Margin" Value="20,30,20,20"/>
+        </Style>
+        <Style Selector="c|Dialog:open /template/ Panel#PART_Overlay">
+          <Setter Property="IsHitTestVisible" Value="False"/>
```

Comment:
IsHitTestVisible=false can be all the time?

Reply:
Hardcoded not in a style

Reply:
it's used for `EnableCancelOnPressed` by handling `PointerPressed` event

Reply:
this is not needed in full screen mode

Reply:
I will do different way to implement dismiss so PART_Overlay is not involved.